UltraStar Play/Assets/Common/Audio/Recording/MicrophonePitchTracker.cs

```diff
@@ -14,12 +14,12 @@ public class MicrophonePitchTracker : MonoBehaviour
     [ReadOnly]
     public string lastMidiNoteName;
 
-    private string micDevice;
-    public string MicDevice
+    private MicProfile microphone;
+    public MicProfile Microphone
```

Comment:
I would prefer if the variable is named the same as the type, i.e., `private MicProfile micProfile` and `public MicProfile MicProfile`.

Reply:
changes this as requested